sims/verilator/Makefile

```diff
@@ -84,13 +83,15 @@ ARIANE_VERILATOR_FLAGS = \
 
 # normal flags used for chipyard builds (that are incompatible with ariane)
 CHIPYARD_VERILATOR_FLAGS = \
+	-Wno-fatal \
 	--assert
 
 VERILATOR_NONCC_OPTS = \
 	--top-module $(VLOG_MODEL) \
 	$(shell if ! grep -iq "module.*ariane" $(build_dir)/*.*v; then echo "$(CHIPYARD_VERILATOR_FLAGS)"; else echo "$(ARIANE_VERILATOR_FLAGS)"; fi) \
 	--output-split 10000 \
 	--output-split-cfuncs 100 \
+	--max-num-width 1048576 \
```

Comment:
This was done to match RC: https://github.com/chipsalliance/rocket-chip/pull/2377